### PR TITLE
Update LRI URL

### PR DESCRIPTION
**Pull request by bdwyer2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ input and outputs a string. The model was trained on the One Billion Word Benchm
 data set. The input to the model is a simple text file, and the output is a string containing the words that are
 predicted to follow. The model has a vocabulary of approximately 800,000 words.
 
-The model files are hosted on [IBM Cloud Object Storage](http://max-assets.s3.us.cloud-object-storage.appdomain.cloud/news-text-generator/1.0/assets.tar.gz). The code in this repository deploys the model as a web service
+The model files are hosted on [IBM Cloud Object Storage](https://max-cdn.cdn.appdomain.cloud/max-news-text-generator/1.0.0). The code in this repository deploys the model as a web service
```
Comment (ptitzler) [inline README.md]: URL = 404. assets.tar.gz?

**Review by ptitzler — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bdwyer2) [reply README.md]: Yes, thanks!

Comment (bdwyer2) [reply README.md]:
```suggestion
The model files are hosted on [IBM Cloud Object Storage](https://max-cdn.cdn.appdomain.cloud/max-news-text-generator/1.0.0/assets.tar.gz). The code in this repository deploys the model as a web service
```

**Review by ptitzler — APPROVED** [review]
LGTM

**Pull request merged**